components/shared/PillTabs.vue

```diff
@@ -39,14 +67,14 @@ const handleTabClick = (value: string) => {
   height: 20px;
 }
 
-.token-price-container {
+.pill-tabs-container {
   gap: 10px;
 
   @media screen and (max-width: 1216px) {
     flex-wrap: wrap !important;
```

Comment:
that's an odd value, can you try and use one of bulma's mixins?
https://bulma.io/documentation/utilities/responsive-mixins/#named-mixins